caddytest/integration/caddyfile_adapt/header.caddyfiletest

```diff
@@ -12,6 +12,9 @@
 	@images path /images/*
 	header @images {
 		Cache-Control "public, max-age=3600, stale-while-revalidate=86400"
+		match {
+			status 200
+		}
```

Comment:
Please add another one which tests having the matcher on a single line, i.e. `match status 200`